internal/provider/resource_gitlab_project_access_token.go

```diff
@@ -90,17 +90,27 @@ var _ = registerResource("gitlab_project_access_token", func() *schema.Resource
 				Type:        schema.TypeInt,
 				Computed:    true,
 			},
+			"access_level": {
+				Description:      fmt.Sprintf("The access level of the associated user_id project membership (cf resource_gitlab_project_membership). Valid values are: %s. Default is `%s`", renderValueListForDocs(validProjectAccessLevelNames), accessLevelValueToName[gitlab.MaintainerPermissions]),
```

Comment:
```suggestion
				Description:      fmt.Sprintf("The access level of the associated user_id project membership (cf resource_gitlab_project_membership). Valid values are: %s. Default is `%s`.", renderValueListForDocs(validProjectAccessLevelNames), accessLevelValueToName[gitlab.MaintainerPermissions]),
```

Nitpick, but our standard has been to end bullet points with a full stop 😄

Reply:
```suggestion
				Description:      fmt.Sprintf("The access level for the project access token. Valid values are: %s. Default is `%s`", renderValueListForDocs(validProjectAccessLevelNames), accessLevelValueToName[gitlab.MaintainerPermissions]),
```

I think that most use cases don't even require nor need to know about the `user_id` generated for the access token (in fact not even required to know that a "user" is even created ...). 
Since *Access Levels* for tokens in GitLab is a common thing I think we can be quite brief on the docs here, too (like the upstream docs are for these cases).